editor/src/clj/editor/workspace.clj

```diff
@@ -339,8 +386,11 @@ ordinary paths."
                                            (set (map resource/proj-path (:added changes)))))) ; no move-source is in :added
          (try
            (let [listeners @(g/node-value workspace :resource-listeners)
-                 total-progress-size (transduce (map first) + 0 listeners)]
-             (load-editor-plugins! workspace (:added changes))
+                 total-progress-size (transduce (map first) + 0 listeners)
+                 added (:added changes)
+                 changed (:changed changes)]
+             (unpack-editor-plugins! workspace (set/union added changed))
+             (load-editor-plugins! workspace (set/union added changed))
```

Comment:
Before loading the plugins, let's unpack them to disc.
* We want to be able to load them easily (e.g. setting LD_LIBRARY_PATH etc)
* We want to give users the opportunity to use their plugins outside of the editor (e.g. creating / modifying data)

Reply:
We can do the `set/union` once here if we add it to the `let` bindings.